README.md

```diff
@@ -1,33 +1,30 @@
 This repository contains resources for configuring kubernetes with calico networking plugin using Solar.
 
-Recommended solar version is `git checkout 1a33a7306d1485f503de967531c87a3b3aff5fcb`.
-
 Express Vagrant setup:
+---------------------
+
+1. `git clone -b stable https://github.com/pigmej/mcpinstall.git && cd mcpinstall`
+2. Get boxes, add boxes(?)
```

Comment:
All vagrant boxes are on atlas, for unofficial fedora you need to download / add it as we did before.